### PR TITLE
DOC: Misc rst syntax fixes

### PR DESCRIPTION
**Pull request by Carreau**
single colon are comments, and not admonitions.

### CODE REVIEW

File: lib/matplotlib/category.py [hunk 1]
```diff
@@ -85,7 +85,7 @@ def axisinfo(unit, axis):
         `~matplotlib.units.AxisInfo`
             Information to support default tick labeling
 
-        .. note: axis is not used
+        .. note:: *axis* is not used
```
Comment (Carreau) [inline lib/matplotlib/category.py:88]: added `*` to match line 42.

Comment (QuLogic) [reply lib/matplotlib/category.py:88]: This should move under the `axis` parameter docs, like on line 42, and not be in the Returns section.

Comment (Carreau) [reply lib/matplotlib/category.py]: Sure. Done.

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
Thanks!

**Pull request merged**